package.json

```diff
@@ -90,6 +90,7 @@
   },
   "dependencies": {
     "@noble/curves": "~1.0.0",
+    "abi-wan-kanabi": "github:haroune-mohammedi/abi-wan-kanabi#feat/starknetjs-integration",
```

Comment:
let's merge your PR in abi-wan first, and then reference the actual package here 🙏

Reply:
Sure

Reply:
If you're OK with the changes, let's merge the abiwan first then this one ?